backend/python/migrations/versions/6bcd31d976d6_create_languages_table.py

```diff
@@ -5,8 +5,8 @@
 Create Date: 2021-12-31 22:23:00.847824
 
 """
-from alembic import op
```

Comment:
just checked - reverting this change does not make the linter unhappy. can you undo this?